x/dex/keeper/msgserver/msg_server_place_orders.go

```diff
@@ -34,6 +34,10 @@ func (k msgServer) transferFunds(goCtx context.Context, msg *types.MsgPlaceOrder
 		return sdkerrors.Wrapf(sdkerrors.ErrUnauthorized, "%s is not allowed to receive funds", contractAddr.String())
 	}
 
+	sender, err := sdk.AccAddressFromBech32(msg.Creator)
```

Comment:
we should do all msg validations in the ValidateBasics of the `message_place_order` so that we don't miss/over check